frontend/e2e/page-objects/InquiryPage.ts

```diff
@@ -0,0 +1,40 @@
+import HelperBase from "./helperBase"
+
+export default class InquiryPage extends HelperBase {
+  async navigateToInquiriesPage() {
+    await this.page.waitForSelector("p.chakra-text.css-0", { timeout: 5000 })
+    await this.page.getByText("Inquiries").click()
+  }
+
+  async openInquiryForm() {
+    await this.page.getByRole("button", { name: "Inquiry" }).click()
+    await this.page.waitForSelector(".chakra-modal__footer", {
+      state: "visible",
+    })
+  }
+
+  async addInquiry(inputText: string) {
+    await this.fillTextByTestId("add-inquiry-text", inputText)
+    await this.assertButtonTextByTestIdAndClick("submit-add-inquiry", "Save")
+  }
+
+  async addInquiry_whenValidInquiryIsAdded_shouldShowInquiryInInquiriesList(
```

Comment:
Meh, sorry for the confusion. So this function should no longer be necessary, right? I.e. you can just call addInquiry directly from the spec?